sdk/nodejs/api/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { QueryTree } from "./api";
+
+  export function queryBuilder(q: QueryTree[]) {
+
+    const args = (item: any) => { 
+      let regex = /\{"[a-zA-Z]+"/ig;
+      
+      return Object.entries(item.args)
```

Comment:
Wondering if this is enough -- see `sdk/go/internal/querybuilder/marshal.go` and ``sdk/go/internal/querybuilder/marshal_test.go`

Reply:
after discussing with Andrea, it's seems to cover a wide range of case. Need a refactoring and do some more tests though.